Jenkinsfile

```
@@ -0,0 +1,3 @@
+common {
+    cron = null
```

```suggestion
    cron = null
    timeoutHours = 2
```

Increase the timeout to 2 hours: https://github.com/confluentinc/jenkins-common/blob/master/vars/jobConfig.groovy#L20